### PR TITLE
META-234 Fix material artwork samples and examples

### PR DESCRIPTION
**Pull request by spacid**
For both the 2D as the 3D sample:
- The METS on the IE level: The IDs of the files in the filesec should be unique.
- The fields birthDate and deathDate in the descriptive metadata should be EDTF according to the spec.

For the 2D sample:
- The premis file objects should have unique identifiers.

For the 3D sample:
- Add the value for the schema namespace.
- Fix link between descriptive metadata and PREMIS on IE level.
- Fix link between descriptive metadata and PREMIS on representation level.
- In general, make sure that the identifiers are unique and are linked correctly between all the preservation metadata.

Remake the bags for both samples.

### CODE REVIEW

**Review by mielvds — APPROVED** [review]
@spacid ziet er goed uit. `edtf:EDTF` is nu overal `edtf:EDTF-level1.` Vernieuw je nog eens de bags en merge je dan?

File: docs/diginstroom/sip/1.1/usecases/video-with-subtitles.md [hunk 1]
```diff
@@ -117,8 +117,8 @@ Note that the identifier is used to link the `dc.xml` file to the corresponding
   <!-- matches the identifier of the premis:object defined in the preservation/premis.xml -->
   <dcterms:identifier>uuid-f58ece94-f050-4b5b-b383-bba83393eaff</dcterms:identifier>
 
-  <dcterms:created xsi:type="edtf:EDTF">the date the episode was created</dcterms:created>
-  <dcterms:issued xsi:type="edtf:EDTF">the date the episode was aired</dcterms:issued>
+  <dcterms:created xsi:type="edtf:EDTF-level1">the date the episode was created</dcterms:created>
```
Comment (spacid) [inline docs/diginstroom/sip/1.1/usecases/video-with-subtitles.md:120]:
```suggestion
  <dcterms:created xsi:type="edtf:EDTF-level1">XXXX</dcterms:created>
```

File: docs/diginstroom/sip/1.1/usecases/video-with-subtitles.md [hunk 2]
```diff
@@ -117,8 +117,8 @@ Note that the identifier is used to link the `dc.xml` file to the corresponding
...
+  <dcterms:issued xsi:type="edtf:EDTF-level1">the date the episode was aired</dcterms:issued>
```
Comment (spacid) [inline docs/diginstroom/sip/1.1/usecases/video-with-subtitles.md:121]:
```suggestion
  <dcterms:issued xsi:type="edtf:EDTF-level1">XXXX</dcterms:issued>
```

File: docs/diginstroom/sip/1.0/usecases/video-with-subtitles.md [hunk 3]
```diff
@@ -115,8 +115,8 @@ Note that the identifier is used to link the `dc.xml` file to the corresponding
   <!-- matches the identifier of the premis:object defined in the preservation/premis.xml -->
   <dcterms:identifier>uuid-f58ece94-f050-4b5b-b383-bba83393eaff</dcterms:identifier>
 
-  <dcterms:created xsi:type="edtf:EDTF">the date the episode was created</dcterms:created>
-  <dcterms:issued xsi:type="edtf:EDTF">the date the episode was aired</dcterms:issued>
+  <dcterms:created xsi:type="edtf:EDTF-level1">the date the episode was created</dcterms:created>
```
Comment (spacid) [inline docs/diginstroom/sip/1.0/usecases/video-with-subtitles.md:118]:
```suggestion
<dcterms:created xsi:type="edtf:EDTF-level1">XXXX</dcterms:created>
```

File: docs/diginstroom/sip/1.0/usecases/video-with-subtitles.md [hunk 4]
```diff
@@ -115,8 +115,8 @@ Note that the identifier is used to link the `dc.xml` file to the corresponding
...
+  <dcterms:issued xsi:type="edtf:EDTF-level1">the date the episode was aired</dcterms:issued>
```
Comment (spacid) [inline docs/diginstroom/sip/1.0/usecases/video-with-subtitles.md:119]:
```suggestion
<dcterms:issued xsi:type="edtf:EDTF-level1">XXXX</dcterms:issued>
```

**Comment by spacid** [comment]
@mielvds Kan je eens naar commit [273ac8a](https://github.com/viaacode/documentation/pull/90/commits/273ac8a8855acea69722768ad61d5cde4bd8abaa) kijken of die goed is?

Gingen we nu "VIAA SP Agent ID" houden als agent type?

File: assets/sip_samples/2D_fa307608-35c3-11ed-9243-7e92631d7d27/data/representations/representation_1/metadata/preservation/premis.xml [hunk 5]
```diff
@@ -23,8 +23,8 @@
       <premis:relationshipType authority="relationshipType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipType/str">structural</premis:relationshipType>
       <premis:relationshipSubType authority="relationshipSubType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType/rep">represents</premis:relationshipSubType>
       <premis:relatedObjectIdentifier>
-        <premis:relatedObjectIdentifierType>VIAA PID</premis:relatedObjectIdentifierType>
```
Comment (mielvds) [inline assets/sip_samples/2D_fa307608-35c3-11ed-9243-7e92631d7d27/data/representations/representation_1/metadata/preservation/premis.xml:26]: bij nader inzien: 
- `VIAA PID` wordt `MEEMOO-PID`
- `VIAA SP Agent ID` wordt `MEEMOO-OR-ID`
- `local_id` wordt `MEEMOO-LOCAL-ID`

Comment (mielvds) [reply assets/sip_samples/2D_fa307608-35c3-11ed-9243-7e92631d7d27/data/representations/representation_1/metadata/preservation/premis.xml:26]: @spacid done in https://github.com/viaacode/documentation/pull/90/commits/4819015d7f573a615ec394c4e688a07d66066762 Check jij?

**Comment by spacid** [comment]
@mielvds Mergen we deze?

**Comment by mielvds** [comment]
👍

**Pull request merged**